webserver/Controllers/RealEstatesController.cs

```diff
@@ -7,7 +7,7 @@
 
 namespace webserver.Controllers;
 
-[Authorize(Roles=Common.BZE_Role+","+Common.Company_Role)]
+[Authorize]
```

Comment:
Every user can see the real states?
Is that what we want?

Reply:
The idea should`ve been that Real Estates are **visible** to everyone, but create-edit-delete operations should`ve been only allowed for the Companies that own the Real Estate, with BZEmployees being allowed full control

I started the API not knowing how to have that finer control so i allowed everyone to get the thing going. But a future P.R will address this issue